test/drlg_l1_test.cpp

```diff
@@ -19,3 +26,134 @@ TEST(Drlg_l1, DRLG_Init_Globals)
 	DRLG_Init_Globals();
 	EXPECT_EQ(dLight[0][0], 0);
 }
+
+void TestCreateL5Dungeon(bool hellfire, int level, int seed, lvl_entry entry)
+{
+	pMegaTiles = std::make_unique<MegaTile[]>(1648);
+
+	MyPlayer->pOriginalCathedral = !hellfire;
+
+	currlevel = level;
+	leveltype = DTYPE_CATHEDRAL;
+
+	CreateL5Dungeon(seed, entry);
+
+	std::string path = paths::BasePath();
+
+	paths::SetPrefPath(path);
+	std::string dunPath;
+	if (hellfire)
+		dunPath = fmt::format("../test/fixtures/hellfire/{}-{}.dun", level, seed);
+	else
+		dunPath = fmt::format("../test/fixtures/diablo/{}-{}.dun", level, seed);
+	auto dunData = LoadFileInMem<uint16_t>(dunPath.c_str());
+	EXPECT_EQ(DMAXX, dunData[0]);
+	EXPECT_EQ(DMAXY, dunData[1]);

```

Comment:
Since it's a dump of the generated level the size should match the levels that we generate. So it's sort of a sanity check in case we expand the size of maps or something like that. I guess ASSERT_EQ is a good choice for this so that we stop the test early if this changes.

```suggestion
	ASSERT_EQ(DMAXX, dunData[0]);
	ASSERT_EQ(DMAXY, dunData[1]);
```